### DISCUSSION TITLE
Columns like in totalcmd

### DISCUSSION THREAD

**Discussion by rromanw** [doublecmd/doublecmd#1257] [Suggestions]
I just tested doublecmd and it seems nice. I would consider using it, but the columns not working like in totalcmd are no go.

1. Columns don't fill the panels by default.
2. When clicking auto fill last/first  column, the resizing works in a wrong way. 

This can be considered UI quirk, but visibility of files is key feature of such software and makes it unusable for me.

Also the rows spacings/height is very big, i would prefer default to look like totalcmd. I'm sure it can be changed in settings somewhere, but I wasn't able to find it quickly.

**Comment by Skif-off** [comment]
> Also the rows spacings/height is very big ... but I wasn't able to find it quickly

Go to "Configuration" in the main menu > Options... > Icons > Icon size > File panel.

Also see ["ExtraLineSpan" from "FilesViews"](https://doublecmd.github.io/doc/en/configxml.html).

**Reply by rromanw** [reply]
Yes, this fixes the icons. 2 bad it isn't default, i think users (newcommers) would appreciate it. The column resizing still not solved for me.
